internal/cmd/providers.go

```diff
@@ -92,13 +94,14 @@ $ infra providers edit okta --client-secret VT_oXtkEDaT7UFY-C3DSRWYb00qyKZ1K1VCq
 # Connect Google to Infra with group sync
 $ infra providers edit google --client-secret VT_oXtkEDaT7UFY-C3DSRWYb00qyKZ1K1VCq7YzN --service-account-key ~/client-123.json --service-account-email hello@example.com --workspace-domain-admin admin@example.com
 `,
-		Args: ExactArgs(1),
+		Args: MaxArgs(5),
```

Comment:
Where do these extra 4 arguments get read? In `RunE` below it seems like we only grab `args[0]` still.

Reply:
This was a mistake, I was thinking of the flags as args, reverted back to 1